tests/nvidiagpu/gpu_suite_test.go

```diff
@@ -32,11 +32,9 @@ var _ = JustAfterEach(func() {
 	reporter.ReportIfFailed(
 		specReport, currentFile, tsparams.ReporterNamespacesToDump, tsparams.ReporterCRDsToDump, clients.SetScheme)
 
-	dumpDir := inittools.GeneralConfig.GetDumpTestReportLocation(currentFile)
-	if dumpDir != "" {
-		artifactDir := fmt.Sprintf("%s/gpu-must-gather", dumpDir)
-		if err := reporter.RunMustGather(artifactDir, 5*time.Minute); err != nil {
-			glog.Errorf("Error running RunMustGather, %v", err)
-		}
+	dumpDir := inittools.GeneralConfig.GetReportPath("gpu-must-gather")
+	artifactDir := fmt.Sprintf("%s/gpu-must-gather", dumpDir)
+	if err := reporter.RunMustGather(specReport, artifactDir, 5*time.Minute); err != nil {
+		glog.Errorf("Error running MustGatherIfFailed, %v", err)
```

Comment:
It's not called `MustGatherIfFailed` anymore. That's why it's not recommended to use function names in logs :) Write in the log message what you wanted to do, e.g. "Failed to collect must-gather for the GPU operator".